frb/dm/prob_dmz.py

```diff
@@ -176,6 +176,32 @@ def grab_repo_grid():
     return sdict
 
 
+
+def grab_chime_repo_grid():
+    """
+    Grab the CHIME grid from the Repository
+
+    Returns:
+        dict: Numpy dict from the npz save file
+    """
+
+    # File
+    CHIME_PDM_z_grid_file = os.path.join(
+        resource_filename('frb', 'data'), 'DM',
```

Comment:
`resource_filename` is deprecated

I now use:

`from importlib.resources import files` or something like that

Reply:
solved